aws/request/http_request_1_4.go

```diff
@@ -1,31 +0,0 @@
-// +build !go1.5
```

Comment:
Because of the above comment, regrettably we still need to keep this version around.